apps/web/src/pages/ApplicantDashboardPage/components/CareerDevelopmentTaskCard.tsx

```diff
@@ -0,0 +1,276 @@
+                    id: "L4UWvU",
+                    description:
+                      "Subtitle explaining career planning expandable withing career development card",
+                  })}
+                >
+                  {intl.formatMessage({
+                    defaultMessage: "Career planning",
+                    id: "E0I0Yj",
+                    description: "Career planning expandable",
+                  })}
+                </Accordion.Trigger>
+                <Accordion.MetaData metadata={careerPlanningMetaData} />
+                <Accordion.Content>
+                  <div
+                    data-h2-display="base(flex)"
+                    data-h2-flex-direction="base(column)"
+                    data-h2-gap="base(x1)"
+                    data-h2-padding-top="base(x.5)"
+                  >
+                    <div>
+                      <p data-h2-font-weight="base(700)">
+                        {intl.formatMessage({
+                          defaultMessage:
+                            "Interest in promotions and lateral moves",
+                          id: "TiPh1V",
+                          description:
+                            "Heading for list of user's interest in employment moves",
+                        })}
+                      </p>
+                      {moveInterests ? (
+                        <MoveInterestsList
+                          moveInterests={moveInterests}
+                        ></MoveInterestsList>
+                      ) : (
+                        <p>
+                          {intl.formatMessage({
+                            defaultMessage: "<red>Missing information</red>",
+                            id: "hI7luh",
+                            description: "Missing information, warning",
+                          })}
+                        </p>
+                      )}
+                    </div>
+                    <div>
+                      <p data-h2-font-weight="base(700)">
+                        {intl.formatMessage({
+                          defaultMessage:
+                            "Types of organizations you'd like to work for",
+                          id: "WvzD/I",
+                          description:
+                            "Heading for list of user's interest in organizations as employers",
+                        })}
+                      </p>
+                      {organizationTypeInterests ? (
+                        <OrganizationTypeInterestsList
+                          organizationTypeInterests={organizationTypeInterests}
+                        ></OrganizationTypeInterestsList>
+                      ) : (
+                        <p>
+                          {intl.formatMessage({
+                            defaultMessage: "<red>Missing information</red>",
```

Comment:
Consider removing the <red> formatting from the string and doing it on the `<p>` element instead so the string can be reused.

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/commit/9368ed0dab4b494a18a55b8fefc2ac88e899f0b8